examples/timeline/other/drag&drop.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Timeline | Drag & Drop</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
+  <script src="../../googleAnalytics.js"></script>
+</head>
+
+<body>
+
+<h1>Timeline Drag & Drop Example</h1>
+
+<p>For this to work, you will have to define your own <code>'dragstart'</code> eventListener on each item in your list of items (make sure that any new item added to the list is attached to this eventListener 'dragstart' handler). This 'dragstart' handler must set <code>dataTransfer</code> - notice you can set  the item's information as you want this way.</p>
+
+<div id="mytimeline" ></div>
+<div>
+  <h3>Items:</h3>
+  <ul class="items">
+    <li draggable="true" class="item">
```

Comment:
These ".items" should get a style with a max-width, a border, a drag-cursor and maybe a icon that shows that they can be dragged (something like "≣")